src/main/resources/view/BrownTheme.css

```diff
@@ -151,6 +151,13 @@
 }
 
 .cell_big_label_event {
+    -fx-font-family: "Segoe UI Semibold";
+    -fx-font-size: 20px;
+    -fx-max-height: 4.5em;  /* Allow exactly 3 lines worth of height for text wrapping*/
```

Comment:
This comment is super helpful to developers, to highlight the fact that `4.5em` is exactly 3 lines worth of height!